### ISSUE TITLE
How to import previous export from maloja ?

### ISSUE THREAD

**Issue by yashaskm11** [krateng/maloja#309]
This is a little long story,
I've been using maloja for a few months now, but recently I had to shutdown my server for a month in September and been scrobbling to lastfm.
I decided to sync scrobbles from lastfm to maloja when I restarted by server in October.
This resulted in double scrobbles (both from lastfm and maloja) of previous ones till September.
But I have a backup of the config before I did the sync.

I continued using the same instance (with double scrobbles), now I want to correct those.
So I exported the json file from the current instance, and filtered the scrobbles to include only the ones that are not double scrobbled (September onwards) using python.

Now I want to run maloja with the backup config (before sync) that has proper scrobbles till September and run `maloja import export.json` in it. Which would solve the double scrobbles.

But I'm not able to do so, and getting an error `File export.json could not be identified as a valid import source.`

Am I in the correct way to solve the above problem ? Any suggestions on how to get around this ?

### **TLDR;**
I want to import the 'maloja_export.json' on another instance and been getting `maloja_export.json is not a valid source` 
I'm using docker version 3.1.5.

**Comment by krateng** [comment]
The file needs just needs to be called `maloja_export_somenumber.json` so it knows what type of import source it is. I added some logic to make it a little better with recognising renamed files in the future.
